lib/Controller/AppSettingsController.php

```diff
@@ -99,6 +102,20 @@ public function setSpreedSettings($stun_server, $turn_server, $turn_server_secre
 				);
 			}
 		}
+		$signaling_server = trim($signaling_server);
+		if ($signaling_server !== '') {
+			$servers = explode('\n', $signaling_server);
+			foreach($servers as $server) {
+				if (!filter_var($server, FILTER_VALIDATE_URL)) {
+					return array('data' =>
+						array('message' =>
+							(string) $this->l10n->t('Invalid signaling server url.')
```

Comment:
cast is not required anymore

Reply:
For consistency I would rather keep this aligned with surrounding code and remove the cast here and in other locations in a follow-up PR.

Reply:
Yeah, same as in the comment below, just ignore for now ;)